paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/EditPaymentMethod.kt

```diff
@@ -178,6 +179,7 @@ private fun Label(
     )
 }
 
+@OptIn(ExperimentalMaterialApi::class)
```

Comment:
Opting in to experimental APIs has caused issues for us in the past. Can we do this with a stable API?

Reply:
Unfortunately not, the base `Button` APIs use the experimental `LocalMinimumInteractiveComponentEnforcement` composition. I would have to rebuild the button from scratch.